website/routes.py

```diff
@@ -1227,6 +1228,13 @@ def make_url_map(app):
         ),
 
         Rule(
+            '/settings/subscribe/',
```

Comment:
Subscriptions can be considered a resource, so a more appropriate URL may be `/api/v1/subscriptions/`. 

```
GET /api/v1/subscriptions/ -> return list of subscriptions
POST /api/v1/subscriptions/ -> create a new subscription
PUT /api/v1/subscriptions/<event>/ -> modify an existing subscription to `<event>`
```